modules/ROOT/pages/lb-ssl-endpoints.adoc

```diff
@@ -0,0 +1,102 @@
+= Configure SSL Endpoints and Certificates
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: cloudhub, runtime manager, arm, load balancing, ssl, two way tls,
+
+A dedicated load balancer must have at least one certificate associated with it. To configure an SSL endpoint for your load balancer to serve to clients, provide a certificate and private key pair for your load balancer to serve to clients. 
+
+A load balancer can have multiple and independent SSL endpoints, each identified by its server certificate common name.
+
+== Requirements
+
+To associate an SSL endpoint, you must provide:
+
+* An unencrypted, PEM-encoded (Privacy-Enhanced Mail) certificate file.
+* A second file containing the private key of your PEM certificate.
++
+The private key file must be _passphraseless_.
+
+== Create a Certificate File and Private Key with Openssl
+
+You can create both an unencrypted, PEM-encoded certificate file and a private key using `openssl`. 
+
+For example, to create a self-signed certificate and corresponding private key:
+
+. Enter:
++
+----
+openssl req -newkey rsa:2048 -nodes -keyout example-com-private.pem -x509 -days 3000 -out example-com-crt.pem
+----
+. When prompted, enter the certificate information.
+
+
+Alternatively, you can pass a certificate information configuration file to the `openssl` command.
+
+For example, this certificate configuration file named `example-com.cfg` defines the DNS domain `example.com` that includes some SAN (Subject Alternate Names) for two Mule applications: `app1.example.com` and `app2.example.com`.
+
+----
+[ req ]
+default_bits       = 2048
+distinguished_name = req_distinguished_name
+req_extensions     = req_ext
+prompt = no
+[ req_distinguished_name ]
+countryName                 = US
+stateOrProvinceName         = California
+localityName               = San Francisco
+organizationName           = MuleSoft
+commonName                 = example.com
+
+[ req_ext ]
+subjectAltName = @alt_names
+[alt_names]
+DNS.1   = app1.example.com
+DNS.2   = app2.example.com
+----
+
+=== Create a Wildcard Certificate
+
+You can create a wildcard certificate, for example, to support any subdomain request to `example.com`, including `app1.example.com` and `app2.example.com`, as well as any future subdomain names.
```

Comment:
Because this is the first sentence in the topic, I am wondering to what "for example" refers? Maybe we need something more like "You might want to create a wildcard certificate to do-this-thing: for example, to support....